src/test/java/com/artipie/rpm/CliTest.java

```diff
@@ -45,4 +48,22 @@ void testWrongArgumentCount() {
             "Expected repository path but got: []"
         );
     }
+
+    @Test
+    void testRunWithCorrectArgument(@TempDir final Path temp) {
+        try {
+            Cli.main(new String[]{"-n=sha256", "-d=sha1", "-f=true", temp.toString()});
+        } catch (final IllegalArgumentException exception) {
+            throw new IllegalArgumentException(exception);
```

Comment:
@marceloamadeu it's redundant to catch this exception and just rethrow it wrapping it in another `IllegalArgumentException`. Just let the method itself throw it and the test will fail, or put some sort of assertion here.